web/src/data/state-icons.json

```diff
@@ -0,0 +1 @@
165],[-278,307],[-419,-373],[-231,-8],[-369,262],[-282,561],[115,99],[-374,127],[-326,352],[-43,287],[-375,515],[62,63],[-317,540],[135,360],[-28,462],[569,76],[69,368],[242,136],[-73,489],[374,652],[190,-98],[89,-352],[283,164],[-104,437],[269,542],[223,-15],[215,388],[174,-177],[333,125],[570,649],[129,41],[25,368],[224,457],[111,719],[179,326],[45,586],[-141,330],[296,145]]],"transform":{"scale":[0.010001000100009998,-0.008850838682731467],"translate":[7.105427357601002e-15,94.24976799431596]}},"wi":{"type":"Topology","objects":{"icon":{"type":"GeometryCollection","bbox":[374783.725617552,-296672.7596767993,850036.6401154302,213021.67400593846],"geometries":[{"type":"MultiPolygon","arcs":[[[0]]]}]}},"arcs":[[[3429,9674],[104,-98],[-217,-283],[-41,-377],[331,174],[498,-290],[285,-68],[154,-360],[1672,-425],[508,-260],[1124,-198],[-62,-212],[429,-135],[161,-170],[-8,-381],[-148,-316],[398,-23],[-146,-315],[264,-234],[-63,-225],[-304,-76],[-270,-526],[-25,-384],[392,231],[249,436],[211,45],[493,662],[156,312],[52,-432],[-118,-59],[-490,-1029],[-157,-608],[18,-368],[-175,-126],[-135,-487],[65,-437],[-323,-938],[47,-537],[244,-493],[-60,-629],[-1683,-35],[-3231,25],[-104,268],[-605,242],[-150,367],[-45,500],[205,258],[-247,212],[-52,924],[-324,504],[-234,59],[-490,393],[-80,292],[-485,241],[-191,256],[-349,38],[-412,450],[106,365],[14,770],[198,323],[-194,280],[-184,23],[42,318],[264,364],[600,256],[94,123],[43,1290],[316,168],[152,-147],[294,11],[1167,352],[209,179],[243,-30]]],"transform":{"scale":[0.009325203752140365,-0.01000100010001],"translate":[3.378643841174238,100]}},"wy":{"type":"Topology","objects":{"icon":{"type":"GeometryCollection","bbox":[-1123282.0641322879,120810.96021315164,807434.5251228312,1635177.627881757],"geometries":[{"type":"Polygon","arcs":[[0]]}]}},"arcs":[[[3134,9807],[2704,-147],[1549,-40],[2506,16],[106,-9616],[-3055,-10],[-2313,72],[-1748,77],[-2883,202],[456,8368],[63,1270],[468,-64],[2147,-128]]],"transform":{"scale":[0.010001000100010001,-0.007844331622306423],"translate":[0,89.21773594572096]}},"dc":{"type":"Topology","objects":{"icon":{"type":"GeometryCollection","bbox":[-8584932.058858203,4694338.7848930545,-8561514.230316386,4721032.765917716],"geometries":[{"type":"MultiPolygon","arcs":[[[0]],[[1]],[[2]]]}]}},"arcs":[[[3147,3896],[317,-289],[-154,-60],[-179,269],[-256,69],[-221,410],[115,18],[378,-417]],[[2893,4409],[-137,9],[-199,441],[236,-42],[100,-408]],[[8974,5399],[1025,-906],[-5086,-4473],[-290,-20],[-268,330],[84,750],[272,969],[246,531],[276,173],[363,486],[397,-52],[415,251],[591,243],[437,537],[-65,144],[-473,-620],[-378,-117],[-478,-282],[-424,83],[-192,-80],[-281,-396],[-319,-143],[-58,673],[-443,405],[-139,21],[366,-434],[83,-251],[-44,-500],[-499,796],[-882,714],[-269,666],[-369,164],[-802,85],[-991,435],[-250,450],[-257,247],[-272,439],[3743,3282],[1966,-1719],[3265,-2881]]],"transform":{"scale":[0.008773577285919407,-0.010001000100010001],"translate":[6.1365003590424365,99.99999999999825]}}}
```

Comment:
Where'd you find this file?  Cool.  😄

Reply:
generated it via https://gist.github.com/mbostock/9821217#file-makefile (with slight modifications to size and simplification settings)